seed_data/seedData.js

```diff
@@ -12,37 +12,49 @@ var freelancers = {
       "_id"         : "5837133bcb98f316ac473853",
       "firstName"   : "John",
       "familyName"  : "Smith",
+      "title"       : "President Candidate",
+      "category"    : "Category A",
+      "description" : "I like potatoes",
+      "urlPic"      : "",
       "address"     : "Caltgadira 116, 6883 Novazzano",
       "email"       : "noneme@icloud.com",
       "phone"       : "032 332 42 72",
       "avgScore"    : 0,
       "price"       : { min : 10, max : 20 },
-      "reviews"     : ["5866a5cc2d6c4804f847ea38", "5866a6162d6c4804f847ea3b"],
-      "tags"        : ["testTag1", "testTag2", "testTag3"]
+      "reviews"     : ["5866a6162d6c4804f847ea3b", "5866a5cc2d6c4804f847ea38"],
+      "tags"        : ["585eaac6f1dc05033ee9ce97"]
     },
     {
       "_id"         : "5837133bcb98f316ac473850",
       "firstName"   : "Isaac",
       "familyName"  : "Asimov",
+      "title"       : "Writer",
+      "category"    : "Category B",
+      "description" : "I like tomatoes",
+      "urlPic"      : "",
       "address"     : "Strickstrasse 64, 8005 Zuerich",
       "email"       : "mschilli@mac.com",
       "phone"       : "043 861 88 44",
       "avgScore"    : 2,
       "price"       : { min : 50, max : 100 },
       "reviews"     : ["5866a5cb2d6c4804f847ea37", "5866a5cb2d6c4804f847ea36"],
-      "tags"        : ["testTag1", "testTag3", "testTag4"]
+      "tags"        : ["585eaac6f1dc05033ee9ce97"]
     },
     {
       "_id"         : "5837133bcb98f316ac473852",
       "firstName"   : "J.R.R.",
       "familyName"  : "Tolkien",
+      "title"       : "Blogger",
```

Comment:
Yes, int the content writer sense - anyway that's up to the user so
Why are categories hard coded - unlike tags?